pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -124,6 +124,10 @@ type CiliumBGPNeighbor struct {
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=4294967295
 	PeerASN int64 `json:"peerASN"`
+	// AuthSecret is the name of the secret to use to fetch a TCP authentication
+	// password for this peer.
+	// +kubebuilder:validation:Optional
+	AuthSecret string `json:"authSecret"`
```

Comment:
Could you follow the k8s API convention and name it `authSecretRef`?

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-of-the-reference-field

Reply:
Sorry, one more k8s styling ask. An optional field should be a pointer.

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-of-the-reference-field